app/screens/home/channel_list/categories_list/categories/index.ts

```diff
@@ -23,12 +23,23 @@ const enhanced = withObservables(
         const currentTeamId = observeCurrentTeamId(database);
         const categories = currentTeamId.pipe(switchMap((ctid) => queryCategoriesByTeamIds(database, [ctid]).observeWithColumns(['sort_order'])));
 
-        const unreadsOnTop = querySidebarPreferences(database, Preferences.CHANNEL_SIDEBAR_GROUP_UNREADS).
+        const unreadsOnTopUserPreference = querySidebarPreferences(database, Preferences.CHANNEL_SIDEBAR_GROUP_UNREADS).
             observeWithColumns(['value']).
             pipe(
-                switchMap((prefs: PreferenceModel[]) => of$(getSidebarPreferenceAsBool(prefs, Preferences.CHANNEL_SIDEBAR_GROUP_UNREADS))),
+                switchMap((prefs: PreferenceModel[]) => of$(getPreferenceValue<string>(prefs, Preferences.CATEGORIES.SIDEBAR_SETTINGS, Preferences.CHANNEL_SIDEBAR_GROUP_UNREADS))),
             );
 
+        const unreadsOnTopServerPreference = observeConfigBooleanValue(database, 'ExperimentalGroupUnreadChannels');
+
+        const unreadsOnTop = combineLatest([unreadsOnTopUserPreference, unreadsOnTopServerPreference]).pipe(
```

Comment:
This adds the `unreadsOnTop` functionality based on the server preference, to add parity with web.

Reply:
Why was this added? Does it need the experimental config? I'm almost sure that got removed

Reply:
This came up in the grooming meeting. Maybe @hmhealey has better context.